src/helpers/vehicleColor.js

```diff
@@ -1,24 +1,21 @@
 import get from "lodash/get";
-import set from "lodash/set";
-import random from "lodash/random";
 
-const vehicleColors = {};
-
-export function getColor(name) {
-  const color = get(vehicleColors, name);
-
-  if (!color) {
-    return setColor(name);
-  }
+const vehicleColors = {
+  BUS: "var(--bus-blue)",
+  TRAM: "var(--green)",
+  RAIL: "var(--purple)",
+  TRUNK: "var(--orange)",
+  SUBWAY: "var(--orange)",
```

Comment:
Although not used in Transitlog (yet!), subway stops are visible on the map nonetheless, so they should also get a color.